src/fri_merkle_statement.rs

```diff
@@ -1,111 +1,66 @@
-use ethers::types::U256;
-use serde::{Deserialize, Serialize};
+use std::sync::Arc;
 
-use crate::serialization::{
-    deserialize_u256_as_number, deserialize_vec_u256_as_number, serialize_u256_as_number,
-    serialize_vec_u256_as_number,
+use ethers::{
+    abi::Address,
+    contract::abigen,
+    core::k256::ecdsa::SigningKey,
+    middleware::SignerMiddleware,
+    providers::{Http, Provider},
+    signers::Wallet,
+    types::U256,
 };
+use serde::{Deserialize, Serialize};
+
+use crate::ContractFunctionCall;
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct FRIMerkleStatement {
-    #[serde(
-        serialize_with = "serialize_u256_as_number",
-        deserialize_with = "deserialize_u256_as_number"
-    )]
```

Comment:
ah I see you're saying we don't need to produce a json anymore because we send directly to Ethereum. Is it for sure that people won't have the need to produce these separately :D?

Reply:
I think it is not a standard to have this type of custom long number value in json file. Initially, the reason we serialized in that way is for communication on the formats. But now we can verify the format directly via the contract interactions. 

The other reason I removed this custom serialization method is for avoiding the confusions in the usage, especially this format is not aligning with the json standard. 

Maybe we can come back to this for a more appropriate serialization/deserialization if there is indeed a need to serialize the contract calldata.